autoload/neomake/makers/ft/scss.vim

```diff
@@ -1,12 +1,30 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#scss#EnabledMakers()
-    return ['scsslint']
+    " Only enable first one found
```

Comment:
This is a matter of architecture, if you prefer doing `return executable('sass-lint') ? ['sass-lint']  : ['scss-lint']` feel free to update the PR to match what you want.  
I just went with the way it was done in `d.vim` since that most suited the linter ecosystem for scss (where you generally only one one linter).

Ideally I'd like a `neomake#makers#Prefer([ { sasslint: 'sass-lint' }, { scsslint: 'scss-lint' } ])` that could be reused across all makers when only one of the given name+executable combinations is found (this would be useful in makers for d.vim, c.vim, python.vim, etc. that all use their own logic/coding style to determine which linter to prefer). Things are not very DRY right now.